### PR TITLE
Create Job processor/mapping from django rest api

### PR DESCRIPTION
**Pull request by kurtw**
This pulls the django job data into elasticsearch for availability in the jinja2 templates.

@dpford @jimmynotjim

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Code looks good. A few questions:

Where does the link to USAJobs come from?
Is the Duties and Departments content in the description objects or will those be new objects?
Where do the specific status/opening types come from (the radio buttons at the bottom of the page)?

Refer to [GHE]/flapjack/careers/issues/5

**Comment by anselmbradford** [comment]
:warning: :warning: :warning:  Remove `static/img/CFPB Analytics.xlsx` and `static/img/Google Analytics - Access request procedure.pdf`. We re-wrote history last week to remove these files.

**Comment by anselmbradford** [comment]
Also, delete `flapjack` branch locally and re-pull it. Sadly you may need to re-create this PR to get the new history right.

**Comment by kurtw** [comment]
I think I may have fixed it. I'm not sure where in the history those things were included so I can't be sure.

**Comment by jimmynotjim** [comment]
Yep, looks like you're in the clear. Let's get this merged so I can keep working on the other careers pages

File: _lib/django_career_processor.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    for career in posts_at_url(url):
+        yield process_career(career)
+
+
+def process_career(career):
+    career['_id'] = career['id']
+    for salary in ['salary_' + m for m in ['max', 'min']
+                   if career['salary_' + m]]:
+        career[salary] = float(career[salary])
+    return {'_index': 'content',
```
Comment (dpford) [inline _lib/django_career_processor.py]: We shouldn't specify the index here, only type, id, and source.

**Pull request merged**